### PR TITLE
platform: use lineage charger

### PR DESCRIPTION
**Pull request by Linux4**
Replaces samsungs proprietary lpm charger. Also this commit fixes some of the selinux denials for health HAL (on beyond1). Please test, works fine for me on beyond1.

### CODE REVIEW

File: platform/product_packages.mk [hunk 1]
```diff
@@ -40,6 +40,10 @@ PRODUCT_PACKAGES += \
 PRODUCT_PACKAGES += \
     Snap
 
+### CHARGER
+PRODUCT_PACKAGES += \
+     lineage_charger_res_images
+
```
Comment (cryptomilk) [inline platform/product_packages.mk]: Isn't that obsolete if we have our own animation?

File: config/charger/animation.txt [hunk 2]
```diff
@@ -0,0 +1,81 @@
+## Google Pixel animation
+
+# animation: num_cycles, first_frame_repeats, animation_file
+animation: 5 1 charger/battery_scale
+
+# percent_display: text_percent_pos_x, text_percent_pos_y, color_r, color_g, color_b, color_a, font_file
+percent_display: c c 255 255 255 255 charger/main_font
```
Comment (cryptomilk) [inline config/charger/animation.txt]: As we can at least easily change the color here, how does it look like if we use:

`percent_display: c c 22 124 128 255 charger/main_font`

Which should be the LineageOS color?

File: platform/product_packages.mk (hunk 1 above)
Comment (Linux4) [reply platform/product_packages.mk]: Yes, forgot to remove it

File: config/charger/animation.txt (hunk 2 above)
Comment (Linux4) [reply config/charger/animation.txt]: No, it's some sort of yellow now

Comment (cryptomilk) [reply config/charger/animation.txt]: isn't that RGBA? Hmm

Comment (cryptomilk) [reply config/charger/animation.txt]: Ah, I think it is rgba(22, 124, 128, 1)

Comment (Linux4) [reply config/charger/animation.txt]: That just makes the percentage disappear ..

Comment (Linux4) [reply config/charger/animation.txt]: Works now with 124 128 22 255

Comment (Linux4) [reply config/charger/animation.txt]: It should be rgba but for some reason it only works with gbra

Comment (Linux4) [reply config/charger/animation.txt]: Maybe related to https://github.com/whatawurst/android_device_samsung_exynos9820-common/blob/lineage-17.1/BoardConfigPlatform.mk#L195 ?

**Pull request merged**